src/controllers/trainee /Controller.ts

```diff
@@ -0,0 +1,85 @@
+import { Request , Response } from 'express';
+
+class TraineeController {
+static instance ;
+static getInstance = () => {
+if ( TraineeController.instance) {
+return TraineeController.instance;
+}
+TraineeController.instance = new TraineeController();
+return TraineeController.instance;
+}
```

Comment:
reomve this unnecessary code snippet instead export the instance of the controller using the new key word